lib/services.rb

```diff
@@ -3,6 +3,10 @@ def self.content_store
     @content_store ||= GdsApi::ContentStore.new(Plek.new.find("content-store"))
   end
 
+  def self.publishing_api
```

Comment:
Instead of defining `publishing_api` here ourselves, we can use the built in [`GdsApi.publishing_api_v2` instance provided by gds-api-adapters](https://github.com/alphagov/gds-api-adapters/blob/d30488d787c98c08702b047f99ce62d42b8a3b1f/lib/gds_api.rb#L142-L153).

It's something [other apps have been moving towards](https://github.com/alphagov/travel-advice-publisher/pull/602).

Reply:
Thanks, will do!